docs/exposing_dns.md

```diff
@@ -61,8 +63,8 @@ apiVersion: v1
 kind: Service
 metadata:
   annotations:
-    service.beta.kubernetes.io/aws-load-balancer-type: nlb # <== tell AWS to use NLB load balancer
-  name: k8gb-coredns-lb
+    service.beta.kubernetes.io/aws-load-balancer-type: nlb
```

Comment:
What was wrong with NLB clarification comment?

Reply:
This is an example of resulted service. Setting and comment moved above to chart setting